ibis/sql/sqlite/compiler.py

```diff
@@ -235,6 +234,23 @@ def _rpad(t, expr):
     return arg + _generic_pad(arg, length, pad)
 
 
+def _row_id(t, expr: ir.Expr):
```

Comment:
is this *only* for sqlite or is it generally available for sql?

Reply:
for postgresql we could use rowid and translate that to `ctid` (I didn't test that yet), about MySQL, I found some discussions but I didn't find a good answer, so not sure if there is a rowid for mysql.